ta/sdp_basic/Android.mk

```diff
@@ -0,0 +1,8 @@
+LOCAL_PATH := $(call my-dir)
+
+ifeq ($(CFG_CACHE_API),y)
+CFLAGS += -DCFG_CACHE_API=y
+endif
```

Comment:
Not needed, `CFG_CACHE_API` should come from `conf.mk` exported by the optee_os build.